WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -178,6 +179,22 @@
            <timeout>800</timeout>
        </test>
 
+       <test>
+           <testName>VHDx_DiskHotPlugUnplug_Multi_SCSI</testName>
+           <preTest>
+               <file>setupscripts\AddVhdxHardDisk.ps1</file>
```

Comment:
This should go as a setupScript otherwise it will give a lot of errors on Remove-VMDvdDrive. All other scripts are using the AddVhdxHardDisk.ps1 as setupScript.